plugins/reolink/src/reolink-api.ts

```diff
@@ -49,7 +49,7 @@ export class ReolinkCameraClient {
     parameters: Record<string, string>;
     tokenLease: number;
 
-    constructor(public host: string, public username: string, public password: string, public channelId: number, public console: Console) {
+    constructor(public host: string, public username: string, public password: string, public channelId: number, public console: Console, public forceToken?: boolean) {
```

Comment:
make this public readonly, forceToken can not be changed

Reply:
done